Lib/asyncio/timeouts.py

```diff
@@ -25,6 +25,7 @@ class _State(enum.Enum):
 
 @final
 class Timeout:
+    """Asynchronous context manager which limits the time spent inside of it."""
```

Comment:
Drop 'of'
```suggestion
    """Asynchronous context manager which limits the time spent inside it."""
```
I'd also appreciate a brief example of how to use it, similar to what you did for TaskGroup.